### DISCUSSION TITLE
Locale-Theme-Menu Demo

### DISCUSSION THREAD

**Discussion by pchemguy** [sqlpage/SQLPage#465] [Show and tell]
The demo is presently hosted in a separate [repo](https://github.com/pchemguy/SQLPage-Demo/tree/main/Locale%20Theme%20Menu)

## Features

- Localization
  The rightmost "Language" menu (only a globe or language symbol is shown) is functional. Select different languages to observe translated menus. Because translation files are real and incomplete, missing items are displayed in English.
- Themes
  The "Theme" menu is functional and permits switching between the included themes (one icon-library-based and one image-based).
- Simulated Login/Logout
  The "Options" menu (only a cog is shown to the left of the "Theme" menu) is mostly not functional. However, pressing the top button imitates the Login/Logout process and adjusts the "Options" and "Help" menu.
- UI Customization
  The button shown on the page toggles language labels in the Language menu.
- Complex flexible menu
- Passing information via the GET URL parameters

![](https://raw.github.com/pchemguy/SQLPage-Demo/main/Locale%20Theme%20Menu/Screenshot/1.png)
![](https://raw.github.com/pchemguy/SQLPage-Demo/main/Locale%20Theme%20Menu/Screenshot/2.png)
![](https://raw.github.com/pchemguy/SQLPage-Demo/main/Locale%20Theme%20Menu/Screenshot/3.png)
![](https://raw.github.com/pchemguy/SQLPage-Demo/main/Locale%20Theme%20Menu/Screenshot/4.png)

**Comment by amrutadotorg** [comment]
Superb work! For my project this should fit ideally. I have 40 languages to deal with.
